packages/gatsby-plugin-sharp/src/scheduler.js

```diff
@@ -6,9 +6,12 @@ const { createProgress } = require(`./utils`)
 
 const toProcess = {}
 let totalJobs = 0
-const q = queue((task, callback) => {
-  task(callback)
-}, 1)
+const q = queue(
+  (task, callback) => {
+    task(callback)
+  },
+  process.env.GATSBY_CLOUD_IMAGE_SERVICE_URL ? Number.MAX_SAFE_INTEGER : 1
```

Comment:
cloud service shouldn't be limited to 1 job at a time, as it just queues jobs - we don't want this internal queue to be bottlenech